src/main/java/week2Task3/Main.java

```diff
@@ -0,0 +1,4 @@
+package week2Task3;
+
+public class Main {
+}
```

Comment:
where is demonstration for AT-2?

Reply:
removed, demo class calls Main, should i refactor name of Main class to Demo ?